### PR TITLE
*: fix decimal fraction value.

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by zimulala** [comment]
LGTM

File: executor/executor_test.go [hunk 1]
```diff
@@ -1325,11 +1325,11 @@ func (s *testSuite) TestDatumXAPI(c *C) {
 	tk.MustExec("create table t (a decimal(10,6), b decimal, index idx_b (b))")
 	tk.MustExec("insert t values (1.1, 1.1)")
 	tk.MustExec("insert t values (2.2, 2.2)")
-	tk.MustExec("insert t values (3.3, 3.3)")
+	tk.MustExec("insert t values (3.3, 2.7)")
 	result := tk.MustQuery("select * from t where a > 1.5")
-	result.Check(testkit.Rows("2.200000 2.2", "3.300000 3.3"))
+	result.Check(testkit.Rows("2.200000 2", "3.300000 3"))
 	result = tk.MustQuery("select * from t where b > 1.5")
```
Comment (shenli) [inline executor/executor_test.go]: Why the query is the same with line 1329?

File: mysql/util_test.go [hunk 2]
```diff
@@ -28,12 +28,18 @@ func TestGetFieldLength(t *testing.T) {
...
+		{TypeNewDecimal, 10},
 	}
 
 	for _, test := range tbl {
 		l := GetDefaultFieldLength(test.Type)
 		if l != test.Length {
 			t.Fatalf("invalid field length %d != %d", l, test.Length)
 		}
+		if test.Type == TypeNewDecimal {
+			if dec := GetDefaultDecimal(test.Type); dec != 0 {
```
Comment (shenli) [inline mysql/util_test.go]: Why not add a independent test case for GetDefaultDecimal?

Comment (coocood) [inline mysql/util_test.go]: The function is too simple.

**Comment by shenli** [comment]
LGTM

**Pull request merged**